src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private static final String ADD_CONFIRMATION_MESSAGE = "Got it. I've added this task:";
+    private static Task[] inputs = new Task[100];
+    private static int index = 0;
+
+    /**
+     * Returns a string with formatted user inputs.
+     *
+     * @param inputs list of user input.
+     * @return Formatted string to print
+     */
+    public static String formatList(Task[] inputs) {
+        String formattedString = HORIZONTAL_LINE;
+        for (int i = 0; i < inputs.length; i++) {
+            if (inputs[i] == null) {
+                break;
+            }
+            formattedString += (i+1) + "." + inputs[i].toString() + "\n";
+        }
+        formattedString += HORIZONTAL_LINE;
+        return formattedString;
+    }
+
+    private static void unmark(String line){
+        int input = Integer.valueOf(line.replace("unmark", "").strip());
+        Task task = inputs[input - 1];
+        task.markAsNotDone();
+        System.out.println(HORIZONTAL_LINE + "OK, I've marked this task as not done yet: \n  "
+                + task + "\n" + HORIZONTAL_LINE);
+    }
+
+    private static void mark(String line) {
+        int input = Integer.valueOf(line.replace("mark", "").strip());
+        Task task = inputs[input - 1];
+        task.markAsDone();
+        System.out.println(HORIZONTAL_LINE + "Nice! I've marked this task as done: \n  "
+                + task + "\n"+ HORIZONTAL_LINE);
+    }
+
+    private static void createTodo(String line){
+        String description = line.replace("todo", "").strip();
+        Todo todo = new Todo(description);
+        inputs[index] = todo;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + todo
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createEvent(String line){
+        String removedCommand = line.replace("event", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/at")).strip();
+        String date = removedCommand.substring(removedCommand.indexOf("/at") + "/at".length()).strip();
+        Event event = new Event(description, date);
+        inputs[index] = event;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + event
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createDeadline(String line) {
+        String removedCommand = line.replace("deadline", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/by")).strip();
```

Comment:
This could be extract into a method and reused (a similar pattern is seen in the createEvent method, where the only difference is the string passed to `.indexOf()`.